### ISSUE TITLE
The language server/VSCode IDE does not recover from error

### ISSUE THREAD

**Issue by mschlaipfer** [dafny-lang/dafny#1990]
When trying the example in #1988 I have to restart the VSCode IDE to start using it again. I couldn't figure out a way to verify other files after the crash without restart.

* Dafny version: 3.5.0.40314
* VSCode plugin: 2.3.0

**Comment by mschlaipfer** [comment]
Sorry if this should have been opened in the VSCode IDE Github project

**Comment by keyboardDrummer** [comment]
For me, changing the file so it no longer triggers the crash and then reopening it resolves the issue.

**Comment by mschlaipfer** [comment]
Yes, avoiding the crash, say by removing everything in the file, and reopening the file makes that file/tab in VSCode usable again. Is there a more graceful way where just removing code from the file without reopening it makes it usable again?

It seems like other files/tabs were not affected (as initially stated).

**Comment by keyboardDrummer** [comment]
I agree. Deleting all text in the file should also completely clear the file state and allow you to continue working again.
